fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMetadata.java

```diff
@@ -62,6 +62,8 @@ default Table getTable(String dbName, String tblName) {
 
     default void createDb(CreateDbStmt stmt) throws DdlException {}
 
+    default void createDb(String clusterName, String dbName, boolean isSetIfNotExists) throws  DdlException {}
```

Comment:
I think the interface can be
```
default void createDb(String dbName) throws DdlException {}
```
the cluster name can be ignored and ifNotExist can be handled in the Executor.

Reply:
I got it. thank you!